database-postgresql/helpers.js

```diff
@@ -0,0 +1,62 @@
+
+const client = require('./connection');
+
+const getPhotos = (trailId, sortOrder, callback) => {
+  let sortOrderStatement = sortOrder ? `ORDER BY upload_date ${sortOrder}` : '';
+
+  let getPhotosPSQLStatement = `SELECT * FROM trailphotos WHERE trail_id = $1` + sortOrderStatement;
+
+  client.query(getPhotosPSQLStatement, [trailId], (err, res) => {
+    if(err) throw err;
```

Comment:
You threw an error. Where is it going? 

Where is the catch in its scope? If there is no catch, what happens?